.stackgenrc.ts

```diff
@@ -76,6 +89,8 @@ new YarnProject(workspace, "core", {
   ],
   scripts: {
     yalc: "npx yalc publish",
+    precompile: "yarn clean",
+    compile: `tsup --dts --out-dir ${buildPath} --entry ./index.ts --entry ./util/logger.ts`,
```

Comment:
Won't two entries create bloated dist files, as they will bundle all deps twice? Need some sort of vendoring, I think?

Reply:
I checked, and you are correct.

Probably need a better tsup defineConfig to make sure that doesn't happen. but this fixes the build for now.

Reply:
Ok, I will resolve, but please backlog this so we don't forget.